docs/metrics.md

```diff
@@ -28,20 +28,33 @@ This is the metrics collection plan for Lockbox's alpha release. It is more of a
 
 Best viewed in something that can render Markdown.
 
-## TL;DR Metrics Priorities for Alpha
+## TL;DR Metrics Implemented for Alpha Thus Far
```

Comment:
FWIW, tweaking this string breaks the TOC links at the top (see line 6 above).

Reply:
@irrationalagent I used the [markdown-toc](https://atom.io/packages/markdown-toc) package to auto-generate and update the table of contents, in case that's easier for you

Reply:
Why do we have a toc here at all? Is there an issue with the toc generated by MkDocs?

Reply:
It was helpful for me to navigate sections when viewing the resulting markdown here on GitHub. I'm not married to it, just thought it was helpful at the time.

...and I now see Travis CI isn't deploying to gh-pages for some reason (stopped 13 days ago). Nice. 😔